MantidQt/MantidWidgets/src/FitPropertyBrowser.cpp

```diff
@@ -169,8 +169,7 @@ void FitPropertyBrowser::init() {
   m_minimizer = m_enumManager->addProperty("Minimizer");
   m_minimizers << "Levenberg-Marquardt"
                << "Levenberg-MarquardtMD"
-               << "More-Sorensen"
```

Comment:
@Anders-Markvardsen So do I understand correctly that removing this is intentional?